File: Python Excel Examples/RangesApiDemo/moveRange.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.ranges_api.RangesApi(configuration)

name = "MoveRange.xlsx"
folder = "Ranges"
storage = ""
sheetName = "Sheet1"
Range = spirecloudexcel.models.Range()
Range.first_row = 1
Range.first_column = 1
Range.row_count = 1
Range.column_count = 1
destRow = 25  # Start value is 0
destColumn = 3  # Start value is 0
api.move_range(name, sheetName, Range, destRow, destColumn, storage = storage, folder = folder)

File: JavaScript Excel Examples/RangesApiDemo/moveRange.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.RangesApi(apiClient);
    
    var name = "moveRange.xlsx";
    var opts = {folder: "input" };
    var sheetName = "Sheet1";
    var range = new SpirecloudExcel.Range();
    range.FirstRow = 1;
    range.FirstColumn = 1;
    range.ColumnCount = 1;
    range.RowCount = 1;
    var destRow = 25;
    var destColumn = 3;
    instance.moveRange(name, sheetName, range, destRow, destColumn, opts,
        function (error, data, response) {
        if (error) throw error;
    });
})();